src/components/Toast/README.md

```diff
@@ -0,0 +1,663 @@
+# Toast
+
+Use MToast to notify users of things.
+
+```vue
+<template>
+	<div>
+		<button @click="openToast">
```

Comment:
Can we add all the possible content to the default scenario? Including icon, and actions? So folks can copy/paste with all the options and just remove what they don't need.

Reply:
I think the first example in the docs of any component should be as simple and basic as possible to make it easy for devs new to the component to be able to grok it quickly. Then the examples get progressively more complicated as they scroll down.

I added 3 more examples to the docs. 2 examples specific to icons and 1 example combining several features show off in the previous one-off examples. I hope this is satisfactory.